packages/eslint-plugin/src/configs/all.json

```diff
@@ -50,6 +50,7 @@
     "@typescript-eslint/no-unnecessary-qualifier": "error",
     "@typescript-eslint/no-unnecessary-type-arguments": "error",
     "@typescript-eslint/no-unnecessary-type-assertion": "error",
+    "@typescript-eslint/no-unused-expressions": "error",
```

Comment:
Can we turn off ESLint's `no-unused-expressions`?

Reply:
ecb3f4ec9b9f8b56896b0ad985547de0e6608381

Reply:
@bradzacher Cool. Thx!!